### DISCUSSION TITLE
loadBalancerClass Kubernetes Support

### DISCUSSION THREAD

**Discussion by infinitydon** [loxilb-io/loxilb#96] [Q&A] [answered]
Hi, I will like to ask if loxilb supports the **_loadBalancerClass_** feature? 

https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class

This is useful in cases where it will be deployed in public cloud platforms that already have a default LoadBalancer controller that responds to LB service-type creation.

**Comment by backguynn** [comment, marked as answer]
Thanks for the nice comments.
Currently, loxilb is not supporting that feature. But we plan to add it in the next release.

**Reply by infinitydon** [reply]
Ok.. Thanks

**Reply by UltraInstinct14** [reply]
@infinitydon 
We are glad to announce support of load-balancer class via [kube-loxilib](https://github.com/loxilb-io/loxilbdocs/blob/main/docs/kube-loxilb.md) . Although in beta, we did test it on AWS EKS kubernetes 1.24 , on-prem K8s/K3s etc. Hope the community finds in valuable.
